src/audio/dai.c

```diff
@@ -74,7 +74,7 @@ static void dai_buffer_process(struct comp_dev *dev, struct dma_sg_elem *next)
 	struct dai_data *dd = comp_get_drvdata(dev);
```

Comment:
@finikorg always explain in the commit message why the change is needed.

Reply:
@dbaluta I have this in the commit message "Fixes issues using void * in arithmetic operations."